src/Test.UnitTests.Sarif.Converters/Test.UnitTests.Sarif.Converters.csproj

```diff
@@ -46,6 +46,11 @@
     <None Remove="TestData\HdfConverter\Inputs\Empty.json" />
     <None Remove="TestData\HdfConverter\Inputs\InvalidResult.json" />
     <None Remove="TestData\HdfConverter\Inputs\ValidResults.json" />
+    <None Remove="TestData\MSBuildConverter\ExpectedOutputs\NoResults.sarif" />
+    <None Remove="TestData\MSBuildConverter\ExpectedOutputs\ValidResults.sarif" />
+    <None Remove="TestData\MSBuildConverter\Inputs\Empty.ERR" />
+    <None Remove="TestData\MSBuildConverter\Inputs\InvalidResult.ERR" />
+    <None Remove="TestData\MSBuildConverter\Inputs\ValidResults.ERR" />
   </ItemGroup>
```

Comment:
fyi I remember Michael once had a comment about removing the whole `<ItemGroup>` instead, if all of the content is not needed, not sure if applies here.

Reply:
thanks removed